redirects/.htaccess_bloggers

```diff
@@ -0,0 +1,432 @@
+# Generated by user_redirect_creator.rb
+
+# Ales
+RewriteRule ^Bloggers/Ales$ /author/ales-justin/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/AlesHome$ /author/ales-justin/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/AlesHome/Page/.*$ /author/ales-justin/ [R=301,L,E=nocache:1]
+RewriteCond %{REQUEST_URI} ^/dirDisplay_d.seam$
```

Comment:
It's the list of a blogger's posts, if you click on the "Browse" button in the upper right of your home page, e.g. at http://in.relation.to/Bloggers/Hardy.